P9Operators/Program.cs

```diff
@@ -0,0 +1,151 @@
+﻿
+// Execise 10
+/*
```

Comment:
Just please next time make sure that the exercise that I actually mean to grade isn't commented out. That's why we create separate projects for separate exercises.